Source/WarpX.H

```diff
@@ -159,14 +176,12 @@ public:
     static short particle_pusher_algo;
     //! Integer that corresponds to the type of Maxwell solver (Yee, CKC, PSATD, ECT)
     static short electromagnetic_solver_id;
-    //! Integer that corresponds to the evolve scheme (explicit, implicit_picard, semi_implicit_picard)
+    //! Integer that corresponds to the evolve scheme (explicit, semi_implicit_em, theta_implicit_em)
     static short evolve_scheme;
-    //! The maximum number of Picard iterations to do each time step
-    static int max_picard_iterations;
-    //! The tolerance for the Picard iteration convergence
-    static amrex::Real picard_iteration_tolerance;
-    //! Flags whether the Picard iterations are required to converge
-    static bool require_picard_convergence;
+    //! The maximum number of iterations used for self-consistent particle update in implicit evolve schemes
+    static int max_particle_iterations;
```

Comment:
Should again make the name for specific
```suggestion
    static int max_particle_iterations_in_implicit_scheme;
```

Reply:
Updated as suggested